app/views/projects/show.html.erb

```diff
@@ -130,7 +130,13 @@
               </div>
           <%end%>
 
-
+          <% if policy(@project).can_feature? %>
+            <%= link_to @project.featured? ? 'Unfeature' : 'Feature' , "" ,class: "card-link", id: 'feature-control', style: "margin-left: 10px" %>
```

Comment:
It is better to make separate forms to keep code cleaner, also it will remove js dependency

Reply:
@tachyons I was working on implementing the first requirement in Issue #412. It asks to implement a button like link to make the circuit featured hence introducing the js dependency to make the ajax request. :sweat_smile:

Reply:
@tachyons this is just a small button and only the admin will be able to see it. This is not a major issue.